app/src/main/res/layout/previous_insurer_bottom_sheet_layout.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I think `layout` might be a bit redundant in the filename, given that it's in `res/layout/*.xml` 😬